### ISSUE TITLE
School Request: UPSA - University of Professional Studies Accra

### ISSUE THREAD

**Issue by justice1k** [blackmann/compa#7]

**Comment by blackmann** [comment]
Nice one. What subdomain will you prefer? It's in the format `<subdomain>.compa.so`. I guess `upsa.compa.so` is fine.

@justice1k 

If you have a custom server to host the project, I can help you. Else, I can host an instance on my server if you want.

**Comment by justice1k** [comment]
Yes, [upsa.compa.so]() and hosting an instance on your server will be great.
